java/api/src/main/java/com/exadel/frs/core/trainservice/controller/EmbeddingController.java

```diff
@@ -51,7 +52,7 @@
     @PostMapping
     public EmbeddingDto addEmbedding(
             @ApiParam(value = IMAGE_WITH_ONE_FACE_DESC, required = true) @RequestParam final MultipartFile file,
-            @ApiParam(value = SUBJECT_DESC, required = true) @RequestParam(SUBJECT) final String subjectName,
+            @ApiParam(value = SUBJECT_DESC, required = true) @Valid @NotBlank(message = "Subject name is empty") @RequestParam(SUBJECT) final String subjectName,
```

Comment:
Please move the message here:
com.exadel.frs.core.trainservice.system.global.Constants